porcupine/plugins/run/no_terminal.py

```diff
@@ -39,7 +40,7 @@ def open_file_with_line_number(path: Path, lineno: int) -> None:
 class NoTerminalRunner:
     def __init__(self, master: tkinter.Misc) -> None:
         # TODO: better coloring that follows the pygments theme
-        self.textwidget = tkinter.Text(master, height=12, state="disabled", name="run_output")
+        self.textwidget = create_passive_text_widget(master, height=12, name="run_output")
```

Comment:
I would prefer a fixed-width (monospace) font, so prevent `create_passive_text_widget` from setting it to `TkDefaultFont` which is not fixed-width. For text widgets, `TkFixedFont` is the default.

```suggestion
        self.textwidget = create_passive_text_widget(master, height=12, name="run_output", font="TkFixedFont")
```